src/ui/view/dataverse/component/pageTemplate/pageTemplate.scss

```
@@ -8,5 +10,36 @@
     display: flex;
     flex-direction: column;
     row-gap: 40px;
+
+    .okp4-dataverse-portal-dataverse-back {
+      @include with-theme {
+        display: flex;
+        align-items: center;
+
+        .okp4-dataverse-portal-dataverse-back-button {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          background: none;
+          border: 0;
+          width: fit-content;
+          padding: 10px 7px;
```

This style can be simplified

```suggestion
        display: flex;
        align-items: center;
        column-gap: 5px;

        .okp4-dataverse-portal-dataverse-back-button {
          all: unset;
          display: flex;
          align-items: center;
```

---

Good point to discuss about. If we define a global style for the button, unsetting all props would give us a different result. In that case, your proposal works.
We can also reset it globally in order to avoid to unset every new button. What do you think @ErikssonJoakim and @lolottetheclash ? Same for other props like `box-sizing`, etc. What are you thoughts about reset? I particularly appreciate this [modern proposal](https://www.joshwcomeau.com/css/custom-css-reset/).

---

Yes we could definitely reset it globally. I think it would be good if we only introduce resets when they are necessary though. 

Seeing as a global reset would implicate all buttons though I would prefer this to be done in a separate refactor PR to better scope the issue

---

Sure, we don't have to add all of these resets. The goal is to avoid resetting default behaviours in many components. Will make a proposal for it, that can be handled in a different PR